### PR TITLE
add student how it works block

### PR DESCRIPTION
**Pull request by Tolik170**
image

### CODE REVIEW

File: tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const mockNavigate = jest.fn()
+
+jest.mock('react-router', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockNavigate
+}))
+
+describe('FindMentorBlock test', () => {
+    beforeEach(() => {
+      render(<StudentHowItWorks />)
```
Comment (tarasSanotskyi) [inline tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js:13]: use renderWithProviders for test router

File: tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { screen, fireEvent, render } from '@testing-library/react'
+import StudentHowItWorks from '~/containers/student-home-page/student-how-it-works/StudentHowItWorks'
+
+const mockNavigate = jest.fn()
+
+jest.mock('react-router', () => ({
+  ...jest.requireActual('react-router-dom'),
```
Comment (tarasSanotskyi) [inline tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js:7]: https://testing-library.com/docs/example-react-router/

**Review by tarasSanotskyi — APPROVED** [review]

File: tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js (hunk 1 above)
Comment (Tolik170) [reply tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js]: done.

File: tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js (hunk 2 above)
Comment (Tolik170) [reply tests/unit/containers/student-home-page/student-how-it-works/StudentHowItWorks.spec.js]: done.

File: src/containers/student-home-page/student-how-it-works/student-how-it-works.styles.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+export const styles = {
+  container: {
+    backgroundColor: '#fff',
```
Comment (LubomyrKashuba) [inline src/containers/student-home-page/student-how-it-works/student-how-it-works.styles.js:3]: use color from palette

Comment (Tolik170) [reply src/containers/student-home-page/student-how-it-works/student-how-it-works.styles.js]: done.

File: src/containers/student-home-page/student-how-it-works/StudentHowItWorks.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        title={ t('studentHomePage.howItWorks.title') }
+        titleStyles={ styles.sectionTitle }
+      />
+
+      <Box sx={ styles.cardsContainer }>
+        { cards }
+      </Box>
+
+      <Button
+        onClick={ redirect } size="extraLarge" variant="contained"
```
Comment (LubomyrKashuba) [inline src/containers/student-home-page/student-how-it-works/StudentHowItWorks.js:60]: use Link from react-router for that

Comment (Tolik170) [reply src/containers/student-home-page/student-how-it-works/StudentHowItWorks.js]: done.

**Review by maarekua — APPROVED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Pull request merged**